criteo_tft/requirements.txt

```diff
@@ -0,0 +1,3 @@
+six==1.10.0
+tensorflow==1.3.0
+tensorflow-transform==0.1.10
```

Comment:
This change LGTM, we just need to make sure we keep this and the setup file in sync to prevent beam execution errors.